packages/button/src/Button.tsx

```diff
@@ -27,14 +30,16 @@ function Button(props: ButtonProps, ref?: ForwardedRef<HTMLButtonElement>) {
     onClick,
     disabled,
     children,
+    href,
     ...rest
-  } = props
+  } = props as ButtonButtonProps
```

Comment:
We can add validator which type it is, so we don't cast it, it may help to use proper args further

https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgEIFcxgPYg13ZAbwChlkZsoBbALmThAE8SBfEk0SWRFfHEABlQAa2JlkACygQY9AM5gooAOZsOYJgAc+mAcgC8aPbn6EAPsYJDRHBLkXJg84SDFGAFACMTIemZAASnofayd5KwFXdwA+ZFCBADpJOHkAeQB3EAAFKGwdKE0PAHJpWWLAoA

Reply:
We have a disabled appearance of a button even if it is a link

Reply:
`ButtonButton` feels like this, maybe better naming can be here:

![](https://www.aquasliv.ru/images/stories/virtuemart/product/241.800.KD.13.jpg)